packages/discovery-provider/src/queries/get_attestation.py

```diff
@@ -30,6 +31,9 @@
 REWARDS_MANAGER_ACCOUNT_PUBLIC_KEY = None
 if REWARDS_MANAGER_ACCOUNT:
     REWARDS_MANAGER_ACCOUNT_PUBLIC_KEY = Pubkey.from_string(REWARDS_MANAGER_ACCOUNT)
+COOLDOWN_CHALLENGE_IDS = ["s", "b"]
+COOLDOWN_DAYS = 7
```

Comment:
can we make this a param on the challenge so that it can be different for staging?